pnpm/src/cmd/complete.ts

```diff
@@ -40,7 +40,7 @@ export async function complete (
       })
       return allProjects
         .filter(({ manifest }) => manifest.name)
-        .map(({ manifest }) => ({ name: manifest.name }))
+        .map(({ manifest }) => ({ name: manifest.name ?? '' }))
```

Comment:
@zkochan How do you want to handle empty names? Filter it out or leave it empty?

Reply:
filter it out